ocs_sample_library_preview/Asset/StreamReference.py

```diff
@@ -0,0 +1,121 @@
+import json
+
+
+class StreamReference(object):
+    """OCS Asset Stream Reference definition"""
+
+    def __init__(self, id: str = None, name: str = None, description: str = None,
+                 stream_id: str = None):
+        """
+        :param id: required
+        :param name: required
+        :param description: not required
```

Comment:
should we move the optional parameter to the end of the list? it would look cleaner, but (I could be wrong) I think having a required parameter after an optional one would prevent a user from being able to pass unnamed arguments while leaving off the description. That would be kind of awkward usage for someone who doesn't always name their arguments.

Reply:
Yeah... I did them in the order they are specified in the source files, but I think you're right.

Reply:
This has been fixed.